giles-eco-tardis/src/main/java/edu/asu/diging/tardis/core/service/impl/ImageFileStorageManager.java

```diff
@@ -0,0 +1,127 @@
+package edu.asu.diging.tardis.core.service.impl;
+
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+
+import javax.imageio.ImageIO;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import edu.asu.diging.gilesecosystem.requests.ICompletedStorageRequest;
+import edu.asu.diging.gilesecosystem.util.files.IFileStorageManager;
+import edu.asu.diging.gilesecosystem.util.properties.IPropertiesManager;
+import edu.asu.diging.tardis.config.Properties;
+import edu.asu.diging.tardis.core.service.IImageFileStorageManager;
+
+@Component
+public class ImageFileStorageManager implements IImageFileStorageManager {
```

Comment:
I don't think this should be its own component. It should be part of the FileService.